templates/wordpress.sql.j2

```diff
@@ -0,0 +1,398 @@
+-- MySQL dump 10.15  Distrib 10.0.29-MariaDB, for debian-linux-gnu (x86_64)
```

Comment:
Forget it, I won't read this.

Reply:
its a database dump after a wordpress installation there is sadly not a better way to do this